src/interpreter/interpreter.cpp

```diff
@@ -47,6 +50,8 @@ struct ExpressionInterpreter : OverriddenVisitor<ExpressionInterpreter, Flow> {
   ExpressionInterpreter(Interpreter& parent) : parent(parent) {}
 
   WasmStore& store() { return InterpreterImpl::getStore(parent); }
+  Frame& frame() { return InterpreterImpl::getFrame(parent); }
```

Comment:
This can be implemented in terms of `store()` rather than having to add a new method to `InterpreterImpl`.